test/unit/transactor_test.rb

```diff
@@ -445,14 +454,14 @@ def setup
   test_authrep 'does not include usages of 0 when it generates a report job' do |_ , method|
     current_time = Time.now
     metric_name = 'some_metric'
-    Metric.save(service_id: @service_id, id: next_id, name: metric_name)
+    Metric.save(service_id: @service_oauth_id, id: next_id, name: metric_name)
```

Comment:
Why did you decide to use an oauth service in these 2 tests specifically?

Reply:
The original tests used oauth services (ie. `@service_id` was indeed a `@service_oauth_id`), so did not want to change existing and unrelated tests.

Reply:
But there are a bunch of tests here and you only changed the last two. I don't see why the tests in this file should use an oauth service.

Reply:
This may be a result of me initially breaking the old/wrong behavior and then trying to keep it to avoid a round of testing whether changing behavior broke any real usage... the RTT might be measured in years.

Reply:
(that is, we can probably drop this diff, will undo this)